spark/src/test/scala/io/ddf/spark/analytics/MLSupporterSuite.scala

```diff
@@ -3,11 +3,12 @@ package io.ddf.spark.analytics
 import io.ddf.DDF
 import io.ddf.ml.IModel
 import io.ddf.spark.{ATestSuite, SparkDDF}
+import org.apache.spark.rdd._
 
 /**
   */
 class MLSupporterSuite extends ATestSuite {
```

Comment:
It seems that it was moved to pAnalytics/ddf_adatao/src/test/scala/com/adatao/ddf/spark/analytics/MLSupporterSuite.scala
so I put the unit test there